Source/Core/InputCommon/ControllerInterface/XInput/XInput.h

```diff
@@ -90,7 +90,7 @@ class Device : public Core::Device
 
 private:
 	XINPUT_STATE m_state_in;
-	XINPUT_VIBRATION m_state_out;
+	XINPUT_VIBRATION m_state_out, m_current_state_out;
```

Comment:
Done. I just did not want to add code that i don't understand, and where i do not know what it could affect. Sometimes it's tiny changes that look harmless that cause hard to investigate issues.